usr/src/uts/common/fs/zfs/sys/dmu_objset.h

```diff
@@ -121,11 +121,14 @@ struct objset {
 
 	/* Protected by os_lock */
 	kmutex_t os_lock;
-	list_t os_dirty_dnodes[TXG_SIZE];
-	list_t os_free_dnodes[TXG_SIZE];
+	struct objset *os_origin_mooch_objset;
```

Comment:
@skiselkov, Thanks for taking a look.  Yeah this was due to mismergeing, I thought I fixed it yesterday but turns out it wasn't included in my push.  Will fix!